code/game/objects/effects/decals/cleanable/chempiles.dm

```diff
@@ -0,0 +1,79 @@
+///HIPPIESTATION MARCHES ON.
+
+GLOBAL_LIST_EMPTY(chempiles)
+/obj/effect/decal/cleanable/chempile
+	name = "chemicals"
+	desc = "An indiscernible mixture of chemicals"
+	icon = 'icons/effects/32x32.dmi'
+	icon_state = "chempile"
+	mergeable_decal = FALSE
+
+/obj/effect/decal/cleanable/chempile/examine(mob/user)
+	. = ..()
+	to_chat(user, "It contains:")//this is so fucking funny I don't want to fix it
```

Comment:
why not replace every to_chat in examine with `. += "string"` like... every other examine does it?

Reply:
//this is so fucking funny I don't want to fix it